runtime/drive_handler.go

```diff
@@ -172,6 +172,11 @@ func (h *StubDriveHandler) Release(
 	return nil
 }
 
+func (h *StubDriveHandler) IsReserved(requestCtx context.Context, id string) bool {
+	_, ok := h.usedDrives[id]
+	return ok
+}
```

Comment:
Possible solution for #630. I would like to split this change and review separately from this PR.